std/protocols/lookup.asm

```diff
@@ -51,21 +54,29 @@ let compute_next_z: Fp2<expr>, Fp2<expr>, Fp2<expr>, Constr, expr -> fe[] = quer
     unpack_ext_array(res)
 };
     
-/// Adds constraints that enforce that rhs is the lookup for lhs
+/// Transfroms a single lookup constraint to identity constraint, challenges and
+/// higher-stage witness columns.
+/// Use this function if the backend does not support lookup constraints natively.
+/// WARNING: This function can currently not be used multiple times since
```

Comment:
How can we refactor this code in the future, so we can use the lookup function multiple times? If we give alpha and beta as parameters, we could've used multiple times